cmd/talosctl/cmd/mgmt/validate.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/talos-systems/talos/internal/app/machined/pkg/runtime"
 	"github.com/talos-systems/talos/pkg/cli"
-	"github.com/talos-systems/talos/pkg/config"
+	"github.com/talos-systems/talos/pkg/config/configloader"
```

Comment:
super duper nit: Since this is already namespaced with "`config`", it might be better to call this package just "`loader`", for the same reasons that (some linter I can't remember) doesn't allow types to have the same name as the package name (e.g. `thing.Thing`).